### PR TITLE
Fixes DOC-1891

### PR DESCRIPTION
**Pull request by lamagnifica**
@lou-wang @nasthagiri @mhoeber please review. Updates the descriptions for the Video Start/Stop Time fields on the Video Component Advanced page to indicate the difference in functionality between browsers and the mobile app.

### CODE REVIEW

**Comment by lou-wang** [comment]
:+1:

File: en_us/shared/building_and_running_chapters/creating_content/create_video.rst [hunk 1]
```diff
@@ -361,19 +361,19 @@ The following options appear on the **Advanced** tab in the video component.
...
       - The time you want the video to stop if you do not want the entire video
-        to play. Formatted as HH:MM:SS. The maximum value is 23:59:59.
+        to play. Use HH:MM:SS format. The maximum value is 23:59:59.
 
-        Note that YouTube offers an option to play the entire video,
-        regardless of the specified stop time. When possible, cut your video
-        files to the desired length before you upload them.
+        .. note:: Learners who download and play the video in the mobile 
+         app see the entire video file. Only videos that play in a browser
+         start at the entered stop time.
```
Comment (nasthagiri) [inline en_us/shared/building_and_running_chapters/creating_content/create_video.rst]: typo? s/start/end

**Comment by nasthagiri** [comment]
:+1: pending typo is fixed.

**Comment by mhoeber** [comment]
:+1:

**Pull request merged**